build.sbt

```diff
@@ -10,34 +10,75 @@ scmInfo in ThisBuild := Some(ScmInfo(
   url("https://github.com/precog/quasar-destination-avalanche"),
   "scm:git@github.com:precog/quasar-destination-avalanche.git"))
 
-val DoobieVersion = "0.8.8"
+//val DoobieVersion = "0.8.8"
+val DoobieVersion = "0.9.0"
+
+lazy val buildSettings = Seq(
+  logBuffered in Test := githubIsWorkflowBuild.value
+  // libraryDependencies ++= Seq(
+  //   "org.slf4s" %% "slf4s-api" % "1.7.25",
+  //   "org.tpolecat" %% "doobie-core" % DoobieVersion,
+  //   "org.tpolecat" %% "doobie-hikari" % DoobieVersion,
+  //   "io.argonaut" %% "argonaut" % "6.3.0-M2",
+  //   "org.typelevel" %% "cats-core" % "2.1.0",
+  //   "com.precog" %% "async-blobstore-core" % managedVersions.value("precog-async-blobstore"),
+  //   "org.specs2" %% "specs2-core" % "4.8.3" % Test
+  // )
+  )
 
 // Include to also publish a project's tests
 lazy val publishTestsSettings = Seq(
   Test / packageBin / publishArtifact := true)
 
+lazy val assemblySettings = Seq(
+  assemblyMergeStrategy in assembly := {
+    case PathList("META-INF", "io.netty.versions.properties") => MergeStrategy.first
+    case x =>
+      val oldStrategy = (assemblyMergeStrategy in assembly).value
+      oldStrategy(x)
+  }
+)
+
+lazy val commonSettings = buildSettings ++ publishTestsSettings ++ assemblySettings
+
 lazy val root = project
   .in(file("."))
   .settings(noPublishSettings)
-  .aggregate(core)
+  .settings(commonSettings)
+  .aggregate(core, azure, s3)
   .enablePlugins(AutomateHeaderPlugin)
 
 lazy val core = project
   .in(file("core"))
-  .settings(name := "quasar-destination-avalanche")
+  .settings(commonSettings)
+  .settings(name := "quasar-destination-avalanche-core")
   .settings(
     performMavenCentralSync := false,
     publishAsOSSProject := true,
+    libraryDependencies ++= Seq(
+     "io.argonaut" %% "argonaut" % "6.3.0-M2",
+     "org.typelevel" %% "cats-core" % "2.1.0", 
+     "org.specs2" %% "specs2-core" % "4.8.3" % Test)
+  )
```

Comment:
Minor nit: collapse these two lines. We don't generally have dangling closing parentheses in this fashion.